alvr/server_openvr/cpp/alvr_server/alvr_server.cpp

```diff
@@ -94,6 +96,15 @@ class DriverProvider : public vr::IServerTrackedDeviceProvider {
         VR_INIT_SERVER_DRIVER_CONTEXT(pContext);
         InitDriverLog(vr::VRDriverLog());
 
+        if (this->early_hmd_initialization) {
+            auto hmd = new Hmd();
+            // Note: we disable awaiting for Acivate() call. That will only be called after
+            // IServerTrackedDeviceProvider::Init() (this function) returns.
+            hmd->register_device(false);
+            this->hmd = std::unique_ptr<Hmd>(hmd);
+            this->tracked_devices.insert({ HEAD_ID, this->hmd.get() });
```

Comment:
This is supposed to mean we do do the active call, waiting for success is just delayed, right?